.ci/clusters/values_runner_images.yaml

```diff
@@ -0,0 +1,138 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+volumes:
+  persistence: false
+
+# disabled AntiAffinity
+affinity:
+  anti_affinity: false
+
+images:
+  zookeeper:
+    repository: streamnative/pulsar-all
+    tag: 2.7.1
```

Comment:
Set this tag to a variable?

Reply:
I don't think helm `values.yaml` support variable in this way, since the `values.yaml` is a variable collection pass to helm template. But the values can be overwritten by using `helm install --set images.zookeeper.tag=2.7.2`